Display number of likes in a user's own comments

meredithcat opened the pull request:
## Description

**Issue:** #512

I added a non-clickable like icon to the user's own inline comments, if the comment has at least one like.

I thought it looked too busy to put the icon side-by-side with the edit/delete icons, so I placed it above. It was a bit tricky to get the CSS right, and there might be a better way that doesn't result in so many new CSS classes.

Journaly.com profile url (include if you'd like the "code contributor" badge): https://journaly.com/dashboard/profile/8654

## Subtasks

- [x] I have added this PR to the Journaly Kanban project ✅

### Deployment Checklist

- [ ] 🚨 Publish new j-db-client version and update in both `web` and `j-mail`
- [ ] 🚨 Deploy migs to stage
- [ ] 🚨 Deploy code to stage
- [ ] 🚨 DEPLOY `j-mail` to stage
- [ ] 🚨 Deploy migs to prod
- [ ] 🚨 Deploy code to prod
- [ ] 🚨 DEPLOY `j-mail` TO PROD

## Screenshots

Short comment with likes:
![Screen Shot 2021-07-30 at 9 41 41 PM](https://user-images.githubusercontent.com/351124/127728872-3153d865-a5b8-45b7-a11c-5876cc040679.png)

Long comment with likes:
![Screen Shot 2021-07-30 at 9 41 25 PM](https://user-images.githubusercontent.com/351124/127728873-c7e7f438-b862-4ac5-aae5-7ce599c05509.png)

A comment without any likes appears the same as before:
![Screen Shot 2021-07-30 at 9 41 50 PM](https://user-images.githubusercontent.com/351124/127728871-41eef0cd-b28d-4bff-82ec-e1eae8c96cb9.png)

robin-macpherson reviewed (changes requested):
Great work! 😄

Still looking through the code and testing but as some early feedback, I like the approach of placing the icon above as you did 👏🏼 One change I'd like to request is for us to simply display the thanks count always, even if it's 0. I can totally see the reason for not showing it but I'd like to keep things as straight-forward and visually consistent as possible. For example we do show 0 likes when not in edit mode and I just want to make sure it's perfectly clear to the user.

Can't wait to get this one through! 🏁 . . . 🦔💨🍃

meredithcat commented:
> One change I'd like to request is for us to simply display the thanks count always, even if it's 0. I can totally see the reason for not showing it but I'd like to keep things as straight-forward and visually consistent as possible. For example we do show 0 likes when not in edit mode and I just want to make sure it's perfectly clear to the user.

Done, plus I added a tooltip to explain the purpose of the icon. I added the tooltip text to the i18n `en` file, but I don't speak German or Spanish well enough to translate! How should I proceed?

robin-macpherson commented:
> Done, plus I added a tooltip to explain the purpose of the icon. I added the tooltip text to the i18n en file, but I don't speak German or Spanish well enough to translate! How should I proceed?

Thanks for adding the tooltip 🙂

We actually have some exciting translation tools that we built now which can be found [here](http://translations-website.s3-website.us-east-2.amazonaws.com/), so unless you speak a language natively or are really confident your translation is correct and natural, it's best to actually leave the translations out so that our translation tools pick those up as missing and community translators can see them flagged as needing translation. We've been testing them out carefully with just a couple of people and languages but will soon roll the translation tools out more publicly 😊

robin-macpherson reviewed (approved):
Looks great! Thanks again so much for tackling this and for being very thorough 🎈🦔
